### PR TITLE
Fix InAuth in entities

### PR DESCRIPTION
**Pull request by csabapalfi**
Fix [InAuth](https://www.inauth.com/) to entities. 

Most sites use their product called [InAuth Inbrowser](https://www.inauth.com/products/inauth-security-platform/inbrowser/). It's a browser fingerprinting solution so I added it as a `"utility"`.

### CODE REVIEW

**Comment by csabapalfi** [comment]
To confirm that these domains are definitely not a generic CDN you can see how the only files served up from there are the JS files called cc.js and et.js for the InAuth product. e.g.

```sql
SELECT url
FROM `httparchive.summary_requests.2018_09_01_desktop`requests
WHERE req_host = 'www.cdn-net.com' and method = 'GET'
GROUP BY url
```

File: data/entities.json5 [hunk 1]
```diff
@@ -2900,8 +2900,9 @@
     "domains": ["*.cdnplanet.com"]
   },
   {
-    "name": "CDN.net",
+    "name": "InAuth",
     "categories": ["utility"],
+    "homepage": "https://www.inauth.com/"
```
Comment (patrickhulce) [inline data/entities.json5]: Anything we can link to for posterity that shows InAuth's connection to this domain? (WHOIS, article, existing site source, docs, github search, anything works :) )

The owner of the domain is fairly obscured 😄

Comment (patrickhulce) [reply data/entities.json5]: Ah you predicted my question all ready! Would you be willing to throw a screenshot of those query results on this PR and we'll call it a day? :D

Comment (patrickhulce) [reply data/entities.json5]: And just to validate that those URLs are indeed for InAuth, here's the current content of the scripts

image

**Comment by csabapalfi** [comment]
Screenshot 2019-05-17 at 00 51 05

**Comment by csabapalfi** [comment]
Screenshot 2019-05-17 at 00 52 43

File: data/entities.json5 (hunk 1 above)
Comment (csabapalfi) [reply data/entities.json5]: haha, just commented the same thing

**Comment by csabapalfi** [comment]
@patrickhulce Thanks for the quick review! Fixed the problem introduced during conflict resolution.

**Pull request merged**